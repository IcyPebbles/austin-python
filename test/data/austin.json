{
  "$schema": "https://www.speedscope.app/file-format-schema.json",
  "shared": {
    "frames": [
      {
        "name": "_find_and_load",
        "file": "<frozen importlib._bootstrap>",
        "line": 971
      },
      {
        "name": "_find_and_load_unlocked",
        "file": "<frozen importlib._bootstrap>",
        "line": 955
      },
      {
        "name": "_load_unlocked",
        "file": "<frozen importlib._bootstrap>",
        "line": 665
      },
      {
        "name": "exec_module",
        "file": "<frozen importlib._bootstrap_external>",
        "line": 678
      },
      {
        "name": "_call_with_frames_removed",
        "file": "<frozen importlib._bootstrap>",
        "line": 219
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/site.py",
        "line": 79
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/os.py",
        "line": 652
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/_collections_abc.py",
        "line": 725
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/site.py",
        "line": 570
      },
      {
        "name": "main",
        "file": "/usr/lib/python3.6/site.py",
        "line": 557
      },
      {
        "name": "addsitepackages",
        "file": "/usr/lib/python3.6/site.py",
        "line": 345
      },
      {
        "name": "addsitedir",
        "file": "/usr/lib/python3.6/site.py",
        "line": 213
      },
      {
        "name": "addpackage",
        "file": "/usr/lib/python3.6/site.py",
        "line": 174
      },
      {
        "name": "<module>",
        "file": "<string>",
        "line": 1
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/types.py",
        "line": 171
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/functools.py",
        "line": 21
      },
      {
        "name": "<module>",
        "file": "/usr/lib/python3.6/collections/__init__.py",
        "line": 62
      },
      {
        "name": "__new__",
        "file": "/usr/lib/python3.6/abc.py",
        "line": 136
      },
      {
        "name": "find_spec",
        "file": "<frozen importlib._bootstrap_external>",
        "line": 1157
      },
      {
        "name": "_get_spec",
        "file": "<frozen importlib._bootstrap_external>",
        "line": 1129
      },
      {
        "name": "find_spec",
        "file": "<frozen importlib._bootstrap_external>",
        "line": 1245
      },
      {
        "name": "_fill_cache",
        "file": "<frozen importlib._bootstrap_external>",
        "line": 1285
      },
      {
        "name": "_bootstrap",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 884
      },
      {
        "name": "_bootstrap_inner",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 916
      },
      {
        "name": "run",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 864
      },
      {
        "name": "keep_cpu_busy",
        "file": "test/target34.py",
        "line": 32
      },
      {
        "name": "<module>",
        "file": "test/target34.py",
        "line": 37
      },
      {
        "name": "start",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 851
      },
      {
        "name": "wait",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 551
      },
      {
        "name": "wait",
        "file": "/usr/lib/python3.6/threading.py",
        "line": 295
      },
      {
        "name": "<module>",
        "file": "test/target34.py",
        "line": 38
      },
      {
        "name": "keep_cpu_busy",
        "file": "test/target34.py",
        "line": 30
      },
      {
        "name": "keep_cpu_busy",
        "file": "test/target34.py",
        "line": 31
      },
      {
        "name": "keep_cpu_busy",
        "file": "test/target34.py",
        "line": 33
      }
    ]
  },
  "profiles": [
    {
      "name": "Time profile of Thread 0:7fdf1b437700",
      "unit": "\u03bcs",
      "startValue": 0,
      "endValue": 393344,
      "samples": [
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          31
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          31
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          31
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          31
        ],
        [
          22,
          23,
          24,
          33
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          25
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          32
        ],
        [
          22,
          23,
          24,
          31
        ],
        [
          22,
          23,
          24
        ],
        [
          22,
          23,
          24
        ],
        [
          22,
          23,
          24
        ]
      ],
      "weights": [
        10092,
        10091,
        10085,
        10088,
        10090,
        10086,
        10093,
        10091,
        10093,
        10084,
        10091,
        10090,
        10090,
        10090,
        10090,
        10062,
        10061,
        10087,
        10091,
        10090,
        10060,
        10091,
        10083,
        10092,
        10090,
        10095,
        10087,
        10058,
        10088,
        10095,
        10094,
        10091,
        10087,
        10089,
        10085,
        10084,
        10093,
        10060,
        10087
      ],
      "type": "sampled"
    },
    {
      "name": "Time profile of Thread 0:7fdf1d34b740",
      "unit": "\u03bcs",
      "startValue": 0,
      "endValue": 423626,
      "samples": [
        [],
        [
          0,
          1,
          2,
          3,
          4,
          5,
          0,
          1,
          2,
          3,
          4,
          6,
          0,
          1,
          2,
          3,
          4,
          7
        ],
        [
          0,
          1,
          2,
          3,
          4,
          8,
          9,
          10,
          11,
          12,
          13,
          0,
          1,
          2,
          3,
          4,
          14,
          0,
          1,
          2,
          3,
          4,
          15,
          0,
          1,
          2,
          3,
          4,
          16,
          17
        ],
        [
          0,
          1,
          2,
          3,
          4,
          8,
          9,
          10,
          11,
          12,
          13,
          18,
          19,
          20,
          21
        ],
        [
          26,
          27,
          28,
          29
        ],
        [
          30,
          31
        ],
        [
          30,
          25
        ],
        [
          30,
          25
        ],
        [
          30,
          33
        ],
        [
          30,
          31
        ],
        [
          30,
          33
        ],
        [
          30,
          33
        ],
        [
          30,
          32
        ],
        [
          30,
          33
        ],
        [
          30,
          32
        ],
        [
          30,
          25
        ],
        [
          30,
          25
        ],
        [
          30,
          25
        ],
        [
          30,
          32
        ],
        [
          30,
          33
        ],
        [
          30,
          33
        ],
        [
          30,
          32
        ],
        [
          30,
          25
        ],
        [
          30,
          32
        ],
        [
          30,
          32
        ],
        [
          30,
          31
        ],
        [
          30,
          33
        ],
        [
          30,
          32
        ],
        [
          30,
          25
        ],
        [
          30,
          33
        ],
        [
          30,
          25
        ],
        [
          30,
          33
        ],
        [
          30,
          25
        ],
        [
          30,
          31
        ],
        [
          30,
          25
        ],
        [
          30,
          32
        ],
        [
          30,
          25
        ],
        [
          30,
          32
        ],
        [
          30,
          25
        ],
        [
          30,
          32
        ],
        [
          30,
          32
        ],
        [
          30
        ],
        [
          30
        ]
      ],
      "weights": [
        0,
        10092,
        10102,
        10088,
        10092,
        10091,
        10085,
        10088,
        10090,
        10086,
        10093,
        10091,
        10093,
        10084,
        10091,
        10090,
        10090,
        10090,
        10090,
        10062,
        10061,
        10087,
        10091,
        10090,
        10060,
        10091,
        10083,
        10092,
        10090,
        10095,
        10087,
        10058,
        10088,
        10095,
        10094,
        10091,
        10087,
        10089,
        10085,
        10084,
        10093,
        10060,
        10087
      ],
      "type": "sampled"
    }
  ],
  "name": "austin.out",
  "exporter": "Austin2Speedscope Converter 0.2.0"
}