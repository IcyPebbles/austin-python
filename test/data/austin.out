Thread 7fdf1d34b740 0
Bad sample 100
Thread 7fdf1d34b740;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/site.py);L79;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/os.py);L652;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/_collections_abc.py);L725 10092
Thread 7fdf1d34b740;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/site.py);L570;main (/usr/lib/python3.6/site.py);L557;addsitepackages (/usr/lib/python3.6/site.py);L345;addsitedir (/usr/lib/python3.6/site.py);L213;addpackage (/usr/lib/python3.6/site.py);L174;<module> (<string>);L1;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/types.py);L171;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/functools.py);L21;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/collections/__init__.py);L62;__new__ (/usr/lib/python3.6/abc.py);L136 10102
Thread 7fdf1d34b740;_find_and_load (<frozen importlib._bootstrap>);L971;_find_and_load_unlocked (<frozen importlib._bootstrap>);L955;_load_unlocked (<frozen importlib._bootstrap>);L665;exec_module (<frozen importlib._bootstrap_external>);L678;_call_with_frames_removed (<frozen importlib._bootstrap>);L219;<module> (/usr/lib/python3.6/site.py);L570;main (/usr/lib/python3.6/site.py);L557;addsitepackages (/usr/lib/python3.6/site.py);L345;addsitedir (/usr/lib/python3.6/site.py);L213;addpackage (/usr/lib/python3.6/site.py);L174;<module> (<string>);L1;find_spec (<frozen importlib._bootstrap_external>);L1157;_get_spec (<frozen importlib._bootstrap_external>);L1129;find_spec (<frozen importlib._bootstrap_external>);L1245;_fill_cache (<frozen importlib._bootstrap_external>);L1285 10088
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10092
Thread 7fdf1d34b740;<module> (test/target34.py);L37;start (/usr/lib/python3.6/threading.py);L851;wait (/usr/lib/python3.6/threading.py);L551;wait (/usr/lib/python3.6/threading.py);L295 10092
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L30 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10085
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10085
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10088
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10088
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10086
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L30 10086
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L30 10093
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10093
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10093
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10093
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10084
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10084
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L30 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10062
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10062
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10061
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10061
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10087
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10087
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L30 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10060
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10060
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L30 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10083
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10083
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10092
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10092
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10090
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10090
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10095
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10095
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10087
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10087
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L30 10058
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L33 10058
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L33 10088
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10088
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10095
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L30 10095
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10094
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10094
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10091
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10091
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L32 10087
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10087
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10089
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10089
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L31 10085
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L32 10085
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864;keep_cpu_busy (test/target34.py);L30 10084
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10084
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864 10093
Thread 7fdf1d34b740;<module> (test/target34.py);L38;keep_cpu_busy (test/target34.py);L31 10093
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864 10060
Thread 7fdf1d34b740;<module> (test/target34.py);L38 10060
Thread 7fdf1b437700;_bootstrap (/usr/lib/python3.6/threading.py);L884;_bootstrap_inner (/usr/lib/python3.6/threading.py);L916;run (/usr/lib/python3.6/threading.py);L864 10087
Thread 7fdf1d34b740;<module> (test/target34.py);L38 10087
